schema/proto3/port.proto

```diff
@@ -63,10 +59,6 @@ message PortConfiguration {
     repeated FixedIp fixed_ips = 12;
     repeated AllowAddressPair allow_address_pairs = 13;
     repeated SecurityGroupId security_group_ids = 14;
-
-    // FIXME: note that for neutron, it sends down more stuff than this
```

Comment:
Let us scope a MVP for basic port programming. We will revisit this part shortly.